### PR TITLE
feat: add `phpunit` completion spec

### PR DESCRIPTION
**Pull request by rubynguyen1510**

### CODE REVIEW

**Comment by rubynguyen1510** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by SimplyMiguel** [comment]
I have read the CLA Document and I hereby sign the CLA

File: src/phpunit.ts [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+    {
+      name: "--coverage-php",
+      description: "Export PHP_CodeCoverage object to file",
+      args: {
+        name: "file",
+        template: "filepaths"
+      },
+    },
+    {
+      name: "--coverage-text=",
```
Comment (fedeci) [inline src/phpunit.ts:39]:
```suggestion
      name: "--coverage-text",
      requiresSeparator: true,
```

**Review by fedeci — APPROVED** [review]
Thanks @rubynguyen1510, can you fix the two highlighted issues, then we can merge it!

**Comment by rubynguyen1510** [comment]
Hello Fig team! I already changed the code as requested.

**Review by fedeci — APPROVED** [review]
Thanks!

**Pull request merged**